lib/00_hello.rb

```diff
@@ -0,0 +1,7 @@
+def hello
+  "Hello!"
```

Comment:
Based on the tests that were provided, this is actually just looking for a return value, not for the phrases to be actually printed out.  Seems counter-intuitive to every "Hello World" exercise I've ever run into but perhaps they're just trying to keep me on my feet :)